apps/mark-scan/backend/src/custom-paper-handler/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const PAPER_HANDLER_STATUS_POLLING_INTERVAL_MS = 100;
+// Slower interval for limited hardware
+export const DEV_PAPER_HANDLER_STATUS_POLLING_INTERVAL_MS = 200;
+export const PAPER_HANDLER_STATUS_POLLING_TIMEOUT_MS = 10_000;
+
+export const BALLOT_IMAGES_PATH = '/tmp';
```

Comment:
Placeholder, I assume? Maybe add a comment explaining what it should be?

Reply:
This was used when calling `interpretSheetAndSaveImages`, but in https://github.com/votingworks/vxsuite/pull/3756/commits/4669a7b630e4cc2e724a7685f0a8930449beefa9 I removed it.

That commit simplifies the state machine to just call `interpretSheet` and remove `combinePageInterpretationsForSheet`. I think this is the right approach because
* mark-scan doesn't ballot images as an output. It saves after scan because image filepaths are needed as an input for interpretation, but we don't need the images to be saved after interpretation, which is what `interpretSheetAndSaveImages` does
* `combinePageInterpretationsForSheet` is a pared down version of logic from VxScan that determines whether the combination of sheet interpretations is valid, since VxScan may be fed a HMPB or BMD ballot (and those ballots may have invalid sheet configurations due to human error). But VxMarkScan (a) scans BMD ballots only and (b) prints and scans before the paper is moved outside the machine, greatly reducing risk of invalid sheet configurations.